src/sock.c

```diff
@@ -234,6 +234,25 @@ struct dnsquery_resourcerecord
 	struct dnsquery_srvrdata rdata;
 };
 
+void netbuf_add_32bitnum(unsigned char *, int, int *, unsigned int);
```

Comment:
I see. Thanks for info.

In this case static functions would be better. The same for the default logger. I close this PR, so open new one if you still want to merge changes with static functions.